src/Service.Host/client/src/components/ChangeRequests/CreateChangeRequest.js

```diff
@@ -80,6 +81,9 @@ function CreateChangeRequest() {
         if (item.changeType === 'BOARDEDIT') {
             return item.boardCode?.length > 0 && item.revisionCode?.length > 0;
         }
+        if (item.changeType === 'REPLACE') {
+            return item.oldPartNumber?.length > 0 && item.newPartNumber?.length > 0;
```

Comment:
is this just to check these strings exist and are not empty? If so can ditch the ?.length > 0 (I think) since empty strings are falsy anyway